docs/js/viz/index.md

```diff
@@ -23,6 +37,8 @@ For example, [chart visualizations](#chart-visualizations) will automatically de
 
 **query**
 
+To create a new visualization you call the `visualize(query)` method on either `Connect` itself or an instance of `Connect`.
+
```

Comment:
Maybe this part can go earlier. Perhaps in the ## Creating visualizations section, it feels a bit too late to be mentioning it here since this is already used in the first example.